packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -49,6 +50,7 @@ export class ExpressionParser {
     this.math.import(this.getFunctionExtensions());
     this.math.import(this.getFunctionOverrides(), { wrap: true, override: true });
     this.validExpressionCache = new Set();
+    this.set('dateUtils', fns); // Support all FNS date modification functions. https://date-fns.org/v2.16.1/docs/Getting-Started
```

Comment:
Small thing, but might be nice to put `dateUtils` into a `getNamespaces()` function (a bit like how we do for `getCustomFunctions()`. Just makes it a little clearer why we're injecting them into the scope, and makes extending this list a little easier 👍